inter/_client.py

```
@@ -58,15 +79,17 @@ def __init__(
     def _get_token(self):
-        response = requests.post(
-            URL.AUTH,
-            data={
-                "client_id": self.client_id,
-                "client_secret": self.client_secret,
-                "scope": " ".join(self.scopes),
-                "grant_type": "client_credentials",
-            },
-            cert=(self.cert_path, self.key_path),
+        response = _ensure_success(
+            requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)